airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -74,6 +74,7 @@ def __init__(
         self.role_name = iam_role_name
         self.s3_glue_logs = 'logs/glue-logs/'
         kwargs['client_type'] = 'glue'
+        kwargs['region_name'] = region_name
```

Comment:
There is a better fix: instead of handling `region_name` argument ourselves, if we remove it as a named parameter we can let the parent class constructor set `self.region_name`.

Reply:
It looks like `../hooks/s3.py` and `glue.py` both extend `AwsBaseHook`, and `s3.py` passed it up to super() to handle the region_name assignment.

Reply:
+1